lab5.h

```diff
@@ -0,0 +1,204 @@
+﻿#include <iostream>
+#include <string>
+using namespace std;
+class Candidate {
+
+private:
+	string name;
+	int count;
+	float percent;
+
+
+
+
+
+
+public:
+	int sum;
+
+
+	string GetName() {
+
+		return name;
+	}
+
+	int GetCount() {
+
+
+		return count;
+	}
+	float GetPercent() {
+
+
+
+		return percent;
+	}
+
+
+	Candidate(string name, int count, float percent) {
+
+		this->name = name;
+		this->count = count;
+		this->percent = percent;
+
+
+
+
+
+	}
+	void Output() {
+		cout << name << count << "\t" << percent << "%" << endl;
+	}
+
+
+
+};
+class Elections {
+
+private:
+
+	string nazar, irena, denys, igor, sasha;
+	
+	int arr[5];
+
+
+public:
+
+
+
+
+
+	int GetArr0() { return arr[0]; }
+
+	int GetArr1() { return arr[1]; }
+
+
+	int GetArr2() { return arr[2]; }
+
+
+	int GetArr3() { return arr[3]; }
+
+	int GetArr4() { return arr[4]; }
+
+
+
+		
+		
+		Elections(string nazar, string irena,string denys, string igor, string sasha) {
```

Comment:
you already set names as strings, there is no need to wright it again

Reply:
it`s a constructor, it should receive strings